app/views/asp/schoolings/_pfmps.html.haml

```
@@ -0,0 +1,35 @@
+- @pfmps.each do |pfmp|
+  .fr-h4
+    PFMP du
+    = l pfmp.start_date
+    au
+    = l pfmp.end_date
+  
+  .fr-table.fr-mb-1w
+    %table
+      %tbody
+        %tr
+          %td Numéro de prestation dossier
+          %td= pfmp.asp_prestation_dossier_id
+
+  = render partial: "pfmps/payment_panel", locals: { pfmp: pfmp }
+
+  - pfmp.payment_requests.joins(:asp_request).each do |payment_request|
+    .fr-table.fr-mx-2w
+      %table
+        %caption.fr-h5.fr-grid-row.fr-grid-row--middle
+          .fr-mr-2w
+            Demande de paiement #
+            = payment_request.id
+          = payment_request.status_badge
+        %tbody
+          %tr
+            %td Montant du paiement
+            %td= number_to_currency pfmp.amount
+          %tr
+            %td Date d'envoi du flux
+            %td
+              - if payment_request.asp_request.sent_at
+                = l payment_request.asp_request.sent_at, format: :long
```

> Je suis pas d'accord. En terme de service, ce serait bête d'empêcher l'asp de voir cette page et de faire leur contrôle juste par ce qu'on a pas de date d'envoi.

Non, c'est une situation qui ne doit jamais se produire et donc il ne faut pas qu'on la traite, au contraire il faut que l'application s'arrête si jamais ça se passe, ce qui nous permet d'éviter tout fonctionnement dans un état imprévu et des bugs derrière. 

fail early: https://stackoverflow.com/a/2807375

PS : on sera notifié par Sentry ou un utilisateur de l'ASP suffisamment vite.

---

En vérité, j'ai surtout fait ça par ce que j'avais le cas avec mes données locales. Mais azy je l'envlève, osef.